hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modalSize {
+    position: absolute;
+    //margin: auto;
+    margin-top: 20px;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 400px;
+    background: #fafafa;
+    boxshadow: 24;
```

Comment:
I don't think this is how box-shadow works, you can remove this line

Reply:
yeeeet